server/middlewares/auth.js

```diff
@@ -11,7 +11,6 @@ const isAuthorized = (req, res, next) => {
         .status(401)
         .send({ statusCode: 401, auth: false, error: 'you are Unauthorized' });
     } else {
-      res.status(200).send({ auth: true, id: token.id });
```

Comment:
I delete this line because it cause error because it cause this error
```
Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:535:11)
    at ServerResponse.header (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/response.js:771:10)
    at ServerResponse.send (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/response.js:170:12)
    at ServerResponse.json (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/response.js:267:15)
    at ServerResponse.send (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/response.js:158:21)
    at serverError (/home/mf/Desktop/Application system/applicants-system/server/controllers/errors/serverError.js:4:6)
    at Layer.handle_error (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/layer.js:71:5)
    at trim_prefix (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/index.js:315:13)
    at /home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/index.js:335:12)
    at next (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/index.js:275:10)
    at Layer.handle [as handle_request] (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/layer.js:97:5)
    at trim_prefix (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/index.js:317:13)
    at /home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/index.js:335:12)
    at next (/home/mf/Desktop/Application system/applicants-system/node_modules/express/lib/router/index.js:275:10)
```
this line send header to client and i can't send response after that if i go to another end point.